pkg/api/routes.go

```diff
@@ -663,7 +663,7 @@ func (rh *RouteHandler) UpdateManifest(response http.ResponseWriter, request *ht
 		return
 	}
 
-	mediaType := request.Header.Get("Content-Type")
+	mediaType := zcommon.GetContentType(request)
```

Comment:
There are two places in pkg/api/routes.go?
666:    mediaType := request.Header.Get("Content-Type")
1219:           if contentType := request.Header.Get("Content-Type"); contentType != constants.BinaryMediaType {

Reply:
I wasn't sure about L1219. In spec it is specified for `/v2/{name}/manifests/{reference} [put]` (L666) that the parameters on the Content-Type header should be ignore , but for `/v2/{name}/blobs/uploads [post]` (L1219) there is no info about that. Do you think I should make the change also for L1219?